packages/ketcher-core/src/application/render/renderers/PolymerBondRenderer/FlexModePolymerBondRenderer.ts

```diff
@@ -119,6 +257,71 @@ export class FlexModePolymerBondRenderer extends BaseRenderer {
     return this.bodyElement;
   }
 
+  private addLineFromTopToRight(): void {
+    this.path = `${this.path} c 0,4.418 3.582,${CORNER_LENGTH} ${CORNER_LENGTH},${CORNER_LENGTH}`;
+  }
+
+  private addLineFromLeftToTop(): void {
+    this.path = `${this.path} c 4.418,0 ${CORNER_LENGTH},-3.582 ${CORNER_LENGTH},-${CORNER_LENGTH}`;
+  }
+
+  private addLineFromBottomToRight(): void {
+    this.path = `${this.path} c 0,-4.418 3.582,-${CORNER_LENGTH} ${CORNER_LENGTH},-${CORNER_LENGTH}`;
+  }
+
+  private addLineFromBottomToLeft(): void {
+    this.path = `${this.path} c 0,-4.418 -3.582,-${CORNER_LENGTH} -${CORNER_LENGTH},-${CORNER_LENGTH}`;
+  }
+
+  private addLineFromLeftToBottom(): void {
+    this.path = `${this.path} c 4.418,0 ${CORNER_LENGTH},3.582 ${CORNER_LENGTH},${CORNER_LENGTH}`;
+  }
+
+  private addLineFromTopToLeft(): void {
+    this.path = `${this.path} c 0,4.418 -3.582,${CORNER_LENGTH} -${CORNER_LENGTH},${CORNER_LENGTH}`;
+  }
+
+  private addLineFromRightToTop(): void {
+    this.path = `${this.path} c -4.418,0 -${CORNER_LENGTH},-3.582 -${CORNER_LENGTH},-${CORNER_LENGTH}`;
+  }
+
+  private addLineFromRightToBottom(): void {
+    this.path = `${this.path} c -4.418,0 -${CORNER_LENGTH},3.582 -${CORNER_LENGTH},${CORNER_LENGTH}`;
+  }
```

Comment:
Looks like we'll need to unify this part with snake mode in future

Reply:
Separated it into helpers functions